setup.py

```diff
@@ -22,31 +22,27 @@
     license='MIT',
     description='Optimizers and filters contributed by the Thumbor community',
     classifiers=[
-        'Development Status :: 4 - Beta',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: MIT License',
-        'Natural Language :: English',
-        'Operating System :: MacOS',
-        'Operating System :: POSIX :: Linux',
-        'Programming Language :: Python :: 2.7',
-        'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
-        'Topic :: Multimedia :: Graphics :: Presentation'
+        "Development Status :: 4 - Beta",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: MIT License",
+        "Natural Language :: English",
+        "Operating System :: MacOS",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3 :: Only",
+        "Topic :: Internet :: WWW/HTTP :: Dynamic Content",
+        "Topic :: Multimedia :: Graphics :: Presentation",
     ],
     packages=['thumbor_plugins', 'thumbor_plugins.optimizers'],
     package_dir={"thumbor_plugins": "thumbor_plugins"},
-    install_requires=[
-        'thumbor',
-    ],
+    install_requires=[],
     extras_require={
         'tests': [
-            'tornado==4.4',
-            'tornado-pyvows>=0.6.0',
-            'coverage',
-            'mock',
-            'colorama',
-            # https://github.com/python-pillow/Pillow/issues/2609
-            'pillow<4.2.0',
-            'nose'
+            "pytest>=6.2.5",
+            'thumbor>=7.0,<8.0',
```

Comment:
I think the new way of specifying versions is `thumbor==7.*,>=7.0.6 (whatever version you want RIGHT NOW). This way you can still say you want greater than 7.0.6 and don't need to worry about the range semantics (>=, < etc). More of a nitpick, but thought of sharing.